recipes/default.rb

```diff
@@ -16,9 +17,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'apt::default'
-include_recipe 'ark::default'
+include_recipe 'mcrouter::_user'
+include_recipe 'mcrouter::_deps_centos' if node['platform'] == 'centos'
+include_recipe 'mcrouter::_deps_ubuntu_16.04' if node['platform'] == 'ubuntu' && node['platform_version'].to_f	== 16.04
 include_recipe 'memcached::default' if node['mcrouter']['local_memcached']
+include_recipe 'mcrouter::double_conversion'
```

Comment:
Looking at `double_conversion.rb` recipe, it calls cmake; however, that's in the dependent recipes ... so what will happen if someone just adds `mcrouter::double_conversion` to a run-list? Will it fail because cmake isn't installed?